assets/scss/_case-studies.scss

```diff
@@ -0,0 +1,142 @@
+// SASS for Case Studies pages go here:
+
+hr {
+  background-color: #999999;
+}
+
+h2 {
+  margin-bottom: 15px !important;
```

Comment:
I do worry about the use of `!important` in our style sheets. Having that delcared can cause unexpected styling behavior and make troubleshooting in the future difficult to track down.

Reply:
Here's a thing! There's no `<h2>` element in https://deploy-preview-20874--kubernetes-io-master-staging.netlify.app/case-studies/ - not even in the iframe.

Reply:
@onlydole @sftim Good pieces to follow up with after merging. 👍